web/custom-plugins/forecast-of-response/src/components/Operations/PhaseView.tsx

```diff
@@ -75,10 +83,53 @@ const SummaryContentWrapper = styled.div`
   justify-content: space-around;
 `
 
-function NewSimulationConfig(props: { defaultVars: any }) {
-  const { defaultVars } = props
-  const [variables, setVariables] = useState<any>(defaultVars)
+function NewSimulationConfig(props: {
+  defaultVars: TVariable[]
+  dottedId: string
+  setVisibleCreateSimScrim: (isVisible: boolean) => void
+  setCreateSimError: (message: string) => void
+}) {
+  const {
+    defaultVars,
+    dottedId,
+    setVisibleCreateSimScrim,
+    setCreateSimError,
+  } = props
+  const [variables, setVariables] = useState<TVariable[]>(
+    defaultVars.map((vari) => ({ ...vari }))
+  )
   const [simConfigName, setSimConfigName] = useState<string>('New Simulation')
+  const { token } = useContext(AuthContext)
+  const dmssAPI = new DmssAPI(token)
+
+  function createSimulation(variables: TVariable[]) {
+    setVisibleCreateSimScrim(false)
+    setCreateSimError('')
+    // Create the simulation entity
+    dmssAPI.generatedDmssApi
+      .explorerAdd({
+        dataSourceId: DEFAULT_DATASOURCE_ID,
+        dottedId: `${dottedId}`,
+        body: {
+          type: Blueprints.SIMULATION_CONFIG,
+          name: simConfigName,
+          variables: variables,
+        },
+      })
+      .catch((e: Error) => {
+        e.json().then((result: any) => {
+          setVisibleCreateSimScrim(true)
+          setCreateSimError('result.message')
```

Comment:
The message is not passed, but the literal string "result.message"

Reply:
Great catch! Fixed